posts/2024-02-26-2023-Rust-Annual-Survey-2023-results.md

```diff
@@ -0,0 +1,331 @@
+    <div style="display: flex; margin-bottom: 10px;">
+        <span>[<a href="../../../images/2024-02-rust-survey-2023/which-statements-apply-to-rust-at-work.png" target="_href_" title="Download chart as PNG">PNG</a>]</span>&nbsp;<span>[<a href="../../../images/2024-02-rust-survey-2023/which-statements-apply-to-rust-at-work.svg" target="_href_" title="Download chart as SVG">SVG</a>]</span>
+    </div>
+</div>
+<!-- Chart which-statements-apply-to-rust-at-work end -->
+
+There are many factors playing into this, but the growing awareness around Rust has likely resulted in the proliferation of resources, allowing new teams using Rust to be better supported.
+
+In terms of technology domains, it seems that Rust is especially popular for creating server backends, web and networking services and cloud technologies.
+
+<!-- Chart technology-domain start -->
+<div>
+    <div class="bar-chart" id="technology-domain" style="height:600px; width:100%;"><noscript>
+<img alt="technology-domain" height="600" src="../../../images/2024-02-rust-survey-2023/technology-domain.png"/>
+</noscript></div>
+    <div style="display: flex; margin-bottom: 10px;">
+        <span>[<a href="../../../images/2024-02-rust-survey-2023/technology-domain.png" target="_href_" title="Download chart as PNG">PNG</a>]</span>&nbsp;<span>[<a href="../../../images/2024-02-rust-survey-2023/technology-domain.svg" target="_href_" title="Download chart as SVG">SVG</a>]</span>&nbsp;<span>[<a href="../../../images/2024-02-rust-survey-2023/technology-domain-wordcloud.png" target="_href_" title="Download open answers as wordcloud PNG">Wordcloud of open answers</a>]</span>
+    </div>
+</div>
+<!-- Chart technology-domain end -->
+
+> You can scroll the chart to the right to see more domains. Note that the Database implementation and Computer Games domains were not offered as closed answers in the 2022 survey (they were merely submitted as open answers), which explains the large jump.
+
+It is exciting to see the continued growth of professional Rust usage and the confidence so many users feel in its performance, control, security and safety, enjoyability, and more!
+
+## Challenges
+
+As always, one of the main goals of the State of Rust survey is to shed light on challenges, concerns, and priorities on Rustaceans’ minds over the past year.
+
+Of those respondents who shared their main worries for the future of Rust (9,374), the majority were concerned about Rust becoming too complex at 43% — a 5pp increase from 2022. 42% of respondents were concerned about a low level of Rust usage in the tech industry. 32% of respondents in 2023 were most concerned about Rust developers and maintainers not being properly supported — a 6pp increase from 2022.
+
+We saw a notable decrease in respondents who were not at all concerned about the future of Rust. 18% in 2023 and 34% in 2022.
+
+Thank you to all participants for your candid feedback which will go a long way toward improving Rust for everyone.
```

Comment:
I wonder if folks think it's necessary to go into more depth on why the survey is showing growing concerns? I leave that up to all of you but just wanted to point out where more analysis might be helpful.

Reply:
Do you mean any specific concerns? The number of people that are not concerned at all has actually decreased.

Reply:
Ah yes sorry I reveresed these in my head during review for some reason. disregard.